source/OptimaJet.DWKit.StarterApplication/Services/BackgroundOrganizationInviteRequestService.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Options;
+using Optimajet.DWKit.StarterApplication.Models;
+using OptimaJet.DWKit.StarterApplication.Repositories;
+using Serilog;
+using static OptimaJet.DWKit.StarterApplication.Utility.EnvironmentHelpers;
+
+namespace OptimaJet.DWKit.StarterApplication.Services
+{
+	public class BackgroundOrganizationInviteRequestService : BackgroundService
+    {
+        protected readonly IBackgroundServiceRepository<OrganizationInviteRequest> requestRepository;
+        protected readonly IBackgroundServiceRepository<EmailQueue> emailQueueRepository;
+        protected readonly BackgroundOrganizationInviteRequestSettings settings;
+        public BackgroundOrganizationInviteRequestService(
+            IOptions<BackgroundOrganizationInviteRequestSettings> options,
+            IBackgroundServiceRepository<OrganizationInviteRequest> requestRepository,
+            IBackgroundServiceRepository<EmailQueue> emailQueueRepository)
+        {
+            this.requestRepository = requestRepository;
+            this.emailQueueRepository = emailQueueRepository;
+            this.settings = options.Value;
+        }
+
+        protected override async Task ExecuteAsync(CancellationToken stoppingToken)
+        {
+            Log.Information("BackgroundOrganizationInviteRequestService is starting.");
+
+            stoppingToken.Register(() => Log.Information("BackgroundOrganizationInviteRequestService is stopping."));
+
+            while (!stoppingToken.IsCancellationRequested)
+            {
+                Log.Information("BackgroundOrganizationInviteRequestService task doing background work.");
+
+                await TryProcessRequests(stoppingToken);
+                // TODO: Determine best way to delay until new records are available
+                await Task.Delay(TimeSpan.FromMinutes(1));
+            }
+
+
+            Log.Information("BackgroundEmailQueueService is stopping.");
+        }
+
+        private async Task TryProcessRequests(CancellationToken stoppingToken)
+        {
+            var requests = await requestRepository.Get().ToListAsync(stoppingToken);
+            foreach (var request in requests)
+            {
+                var emailQueue = new EmailQueue
+                {
+                    // TODO: Query Users for Super Admins
+                    To = settings.SuperAdminEmail,
+                    Subject = "[Scriptoria] Organization Invite Request",
+                    ContentTemplate = "OrganizationInviteRequest",
```

Comment:
Don't have an answer, but the template is used so we can customize and localize.  The subject line is hard coded English.  Maybe at least a TODO comment to point out that we need to address this sometime